tests/client/utils.js

```diff
@@ -4,15 +4,15 @@ Tinytest.addAsync(
   function (test, done) {
     const SettingsCollection = new Meteor.Collection('kadira_settings');
     let ran = false;
-    SettingsCollection.find().observe({added: function (options) {
+    SettingsCollection.find().observe({added (options) {
       if (ran) {
         return;
       }
       ran = true;
 
-      test.equal(!!options.appId, true);
-      test.equal(!!options.endpoint, true);
-      test.equal(!!options.clientEngineSyncDelay, true);
+      test.equal(Boolean(options.appId), true);
+      test.equal(Boolean(options.endpoint), true);
+      test.equal(Boolean(options.clientEngineSyncDelay), true);
```

Comment:
Could you please adjust the config to allow the old style of converting to a boolean?